roseau/load_flow/models/lines/parameters.py

```diff
@@ -81,11 +88,16 @@ def __init__(
             y_shunt:
                 The Y matrix of the line (Siemens/km). This field is optional if the line has no shunt part.
 
-            max_current:
-                The maximum current loading of the line (A). The maximum current is optional, it is
+            max_currents:
+                The maximum currents loading of the line (A). The maximum currents are optional, they are
                 not used in the load flow but can be used to check for overloading.
                 See also :meth:`Line.res_violated <roseau.load_flow.Line.res_violated>`.
 
+                When a scalar value is provided, it creates an array with the same maximum current for each conductor.
+                The scalar value passed is assumed to be the maximum current of each conductor, not the total
+                multi-phase maximum current. To create a different maximum current per conductor, provide a
+                vector of current values with the same length as the number of conductor of the line.
```

Comment:
The "total multi-phase maximum current" doesn't make sense (unlike powers of loads)

```suggestion
                To create a different maximum current per conductor, provide a vector of current values with
                the same length as the number of conductor of the line.
```